client/src/components/Overview.jsx

```diff
@@ -52,7 +52,7 @@ class Overview extends React.Component {
         this.setState({rating: rating});
       });
 
-    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/40344', {
+    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/40349', {
       headers: {
         authorization: token
       }
```

Comment:
You do not have to implement this yet; we will revisit this later.
- Do not use the API urls anymore; use localhost to make GET requests
- Remove headers once you've made the change above; the header is handled in the Server

Reply:
Will address this after. Thanks for the heads up!